### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by XCTR0X** [lenis0012/LoginSecurity-2#83]
Hey whenever i use this plugin im always getting a problem and cant load this plugin.
Can i somehow make this work without getting database error?

`[18:33:33 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.PlayerLocation but not supported by DB Platform - ignored
[18:33:33 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.PlayerProfile but not supported by DB Platform - ignored
[18:33:33 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.ActionEntry but not supported by DB Platform - ignored
[18:33:33 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.PlayerInventory but not supported by DB Platform - ignored
[18:33:33 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.Migration but not supported by DB Platform - ignored`

```
`18:33:41 INFO]: DataSourcePool [LoginSecurityDB] autoCommit[false] transIsolation[SERIALIZABLE] min[2] max[20]
[18:33:41 INFO]: SubClassFactory parent ClassLoader [org.bukkit.plugin.java.PluginClassLoader]
[18:33:41 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.PlayerLocation but not supported by DB Platform - ignored
[18:33:41 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.PlayerProfile but not supported by DB Platform - ignored
[18:33:41 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.ActionEntry but not supported by DB Platform - ignored
[18:33:41 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.PlayerInventory but not supported by DB Platform - ignored
[18:33:41 INFO]: Explicit Identity on com.lenis0012.bukkit.loginsecurity.storage.Migration but not supported by DB Platform - ignored`

`

**Comment by Tzzzt123** [comment]
Please tell the server version so i can try to help you

**Comment by Tzzzt123** [comment]
and tell me what database platform you are using (MySQL or Sqlite)
Then @lenis0012 can fix it faster

**Comment by lenis0012** [comment]
@XCTR0X This is not an error. It can be ignored.
This message will be gone in the 2.2 update,

@Tzzzt123 Thanks for trying to help out =)

**Comment by lenis0012** [comment]
This is fixed as of version 3.0.0
